src/engine/loopingcontrol.cpp

```diff
@@ -452,6 +452,55 @@ void LoopingControl::hintReader(HintVector* pHintList) {
     }
 }
 
+double LoopingControl::getSyncPositionInsideLoop(double dRequestedPlaypos, double dSyncedPlayPos) {
+    // no loop, no adjustment
+    if (!m_bLoopingEnabled) {
+        return dSyncedPlayPos;
+    }
+
+    LoopSamples loopSamples = m_loopSamples.getValue();
+
+    // if the request itself is outside loop do nothing
+    // loop will be disabled later by notifySeek(...)
```

Comment:
"why" is also interesting: 
... because the user has explicitly requested to jump outside the loop.

or similar.